core/src/test/java/com/linecorp/armeria/server/ServiceBindingTest.java

```diff
@@ -42,6 +47,27 @@
 
     private static CountDownLatch accessLogWriterCheckLatch;
     private static CountDownLatch propertyCheckLatch;
+    private static final Queue<Integer> decoratorLog = new ArrayBlockingQueue<>(100);
+
+    private static final class LoggingDecorator implements Function<HttpService, HttpService> {
```

Comment:
How about moving to the end of the file?

Reply:
I generally like to put dependencies first and things that depend on them after? But happy to move it if that's this project's style.

Reply:
We prefer top-down layout for methods and classes. Thanks in advance. :bow: